src/utils/index.ts

```diff
@@ -134,11 +134,15 @@ export async function dialHelper(
     dhtResponse = await (libp2p._dht as any).findPeer(destination, { timeout: DEFAULT_DHT_QUERY_TIMEOUT })
   } catch (err) {
     error(
-      // prettier-ignore
       `Querying the DHT for ${green(destination.toB58String())} failed. Known addresses:\n` +
-      `  ${renderPeerStoreAddresses(libp2p.peerStore.get(destination)?.addresses ?? [])}.\n` +
-      `${err.message}`
+        `  ${renderPeerStoreAddresses(libp2p.peerStore.get(destination)?.addresses ?? [])}.\n`
     )
+
+    if (err instanceof Error) {
+      error(err.message)
+    } else {
+      error(`Non-error message was thrown`, err)
```

Comment:
worth printing the backtrace to learn where it comes from in all these case as for me

Reply:
added console.traces